server/core/pom.xml

```diff
@@ -59,6 +75,14 @@
    </dependencies>
 
    <build>
+      <!-- Workaround for ISPN-8843 -->
+      <extensions>
+         <extension>
+            <groupId>kr.motd.maven</groupId>
+            <artifactId>os-maven-plugin</artifactId>
+            <version>1.4.1.Final</version>
```

Comment:
Version should probably be in the BOM or parent pom.

Reply:
Let's not make this workaround official :)

Reply:
done